### DISCUSSION TITLE
复制增强功能卡住数秒才能使用

### DISCUSSION THREAD

**Discussion by Faith-Mian** [shuax/MouseInc.Settings#72] [Q&A]
环境：win7 64，各硬件配置合格，有ssd，已关闭所有软件包括火绒

选中文字快速按下两次Ctrl+C，此时鼠标可以自由移动，系统完全静止，trafficmonitor系统资源监视器也完全停止，
此时键鼠按下均无响应，无法呼出任务管理器，但会在数秒恢复正常后依次响应各事件。

**Comment by shuax** [comment]
有win7用户反映过相同bug，但是我虚拟机未能重现，只能暂时不用这个功能。

**Reply by Faith-Mian** [reply]
感谢回复，虽然我对这个功能需求不大，但有没有什么我能为mouseinc做的，比如说提供日志之类的？

**Comment by shuax** [comment]
暂时不用，可能改一种写法才行。
